go/bindings/build.go

```diff
@@ -69,6 +71,8 @@ type BuildArgs struct {
 	CaptureDriverFn CaptureDriverFn
 	// Builder of materialization DriverClients
 	MaterializeDriverFn MaterializeDriverFn
+	// Build container name
+	ContainerName string
```

Comment:
I don't think `ContainerName` can be provided on `BuildArgs`. The problem is that a build may actually start _many_ containers, and they of course can't all have the same name. Builds will perform a `Validate` RPC for each capture and materialization that's involved in the build. There can be many of those, as a build can include arbitrary subsets of all the tasks in the control plane.

I think the solution here is to add the `ContainerName` parameter to the `CaptureDriverFn` and `MaterializeDriverFn`. Those functions each return a driver (backed by a running container) for a specific connector, and the task name, if applicable, should be known when those functions are called.

Reply:
Good point on `request.Capture`. I used that instead of `BuildArgs`. 
`ContainerName` is already passed to `CaptureDriverFn` and `MaterializeDriverFn`